src/shared/types/store.ts

```diff
@@ -58,11 +59,12 @@ export type TLoginSession = {
   encryptedPassword: string
 }
 
-export type TAvailableLanguages = 'English' | 'Deutsch' | '简体中文' | '繁體中文'
+export type TAvailableLanguages = 'English' | 'Deutsch' | 'Português Brasileiro' | '简体中文' | '繁體中文'
 
 export type TLanguage = {
   value: string
   label: TAvailableLanguages
+  locale: Locale
```

Comment:
It's necessary to have a migration. I think it's a non-sustainable solution, because you are saving the `date-fns` locale object in our store.